components/atom/skeleton/src/index.scss

```diff
@@ -0,0 +1,59 @@
+@import '~@s-ui/theme/lib/index';
+
+$bg-skeleton: $c-gray-light-3 !default;
+$bg-skeleton-highlight: $c-gray-light-4 !default;
+$bdrs-skeleton: 8px !default;
```

Comment:
I suggest `$bdrs-base` here
you can fin it here: https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_box-style.scss